### PR TITLE
add: Card Component

### PR DESCRIPTION
**Pull request by kavyakunder**
Card Components
card1

Card with dismiss
card2

card with badge
card3

card with text only
card4

card overlay
card5

horizontal card
card6

### CODE REVIEW

File: Components/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  color: var(--white);
+  padding: 10px;
+  top: 10px;
+  left: 0px;
+  border-radius: 0 5px 5px 0;
+}
+
+/*  Text Only */
+.card-text-only {
+  height: 14rem;
```
Comment (junaidshaikh-js) [inline Components/Card/card.css]: always use min-height

File: Components/Card/card.html [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+                  frameborder="0"
+                ></iframe>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+  </body>
+  <script src="index.js"></script>
```
Comment (junaidshaikh-js) [inline Components/Card/card.html]: script tag should be inside body element

**Comment by saiyerniakhil** [comment]
Hey there, the cards look amazing. but somewhat I feel that the shadow is dominating the card.

**Review by shivampandey0 — COMMENTED** [review]
try to use variables for units.

File: Components/Card/card.html (hunk 2 above)
Comment (kavyakunder) [reply Components/Card/card.html]: removed script tag.

**Comment by kavyakunder** [comment]
> Hey there, the cards look amazing. but somewhat I feel that the shadow is dominating the card.

I changed it to a lower value.

**Pull request merged**